api-models/src/main/thrift/appsRendering.thrift

```diff
@@ -0,0 +1,17 @@
+include "v1.thrift"
+
+namespace scala com.gu.mobile.ssr.models
+
+struct appsRendering {
+
+    1: required v1.Content content
+
+    2: optional i32 commentCount 
+}
+
+struct appsRenderingBlocks {
```

Comment:
Is this for the client-side requests?

Reply:
yep we can rename the type, but this is for liveblog fetching. You would be able to setState using this format and render the list of blocks.